src/planscape/plan/views.py

```diff
@@ -278,6 +278,20 @@ def update_project(request: HttpRequest) -> HttpResponse:
         return HttpResponseBadRequest("Ill-formed request: " + str(e))
 
 
+@csrf_exempt
+def list_projects_for_plan(request: HttpRequest) -> HttpResponse:
+    try:
+        assert isinstance(request.GET['plan_id'], str)
+        plan_id = request.GET.get('plan_id', "0")
+
+        user = _get_user(request)
+
+        projects = Project.objects.filter(owner=user, plan=plan_id)
```

Comment:
Should this say plan=int(plan_id)?

Reply:
I think the implicit conversion works, but I'll make it explicit